package.json

```diff
@@ -80,6 +80,7 @@
     "husky": "^7.0.2",
     "jest": "^27.2.0",
     "jsfuzz": "^1.0.15",
+    "lint-staged": "^12.3.8",
```

Comment:
If you are using npx why is the dep needed?

Reply:
- [x] Fixed in 10614b0